doc/Preferences.md

```diff
@@ -1,19 +1,21 @@
 Theia has a preference service which allows modules to get preference values, contribute default preferences and listen for preference changes.
 
-Preferences can be saved in the root of the workspace under `.theia/settings.json` or under `$HOME/.theia/settings.json` on Linux systems. For Windows systems, the user settings will by default be in the `%USERPROFILE%/.theia/settings.json` (something like `C:\Users\epatpol\.theia/settings.json`)
+Preferences can be saved in the root of the workspace under `.theia/settings.json` or under `$HOME/.theia/settings.json` on Linux systems. For Windows systems, the user settings will by default be in the `%USERPROFILE%/.theia/settings.json` (something like `C:\Users\epatpol\.theia/settings.json`). Note that preferences defined in the workspace will override any preferences defined in the user directory, so it's possible to have different preferences per workspaces.
```

Comment:
If you are planning to apply any further changes, please update the Windows path from `C:\Users\epatpol\.theia/settings.json` to `C:\Users\epatpol\.theia\settings.json`. It is a typo in the path, right?

Reply:
Good call, let me fix it!